resources/puppetlabs/lein-ezbake/staging-templates/Dockerfile.mustache

```diff
@@ -0,0 +1,12 @@
+FROM {{{base-image}}}
+
+COPY *.jar /src/
+COPY Dockerfile /Dockerfile
+COPY ext/config/docker /config
+
+VOLUME /ssl
+
+{{{expose-ports}}}
+
+ENTRYPOINT java -Djava.security.egd=/dev/urandom -XX:OnOutOfMemoryError="kill -9 %p" -cp "{{{classpath}}}" clojure.main -m puppetlabs.trapperkeeper.main --config /config/conf.d --bootstrap-config /config/bootstrap.cfg
```

Comment:
*Really, really* minor: might it be worthwhile to parameterize ```puppetlabs.trapperkeeper.main``` for the case of debugging/profiling? I can imagine a rather cornery-case where having the option to swap it for another method could be useful.

However, this comes with complete ignorance of the actual code inside puppetlabs.trapperkeeper.* on my part.

Reply:
You can override the entrypoint on the CLI or in another Dockerfile, which provides some ability to debug.